chaincode/ledger_db.go

```diff
@@ -176,6 +177,44 @@ func (db *LedgerDB) GetIndexKeys(index string, attributes []string) ([]string, e
 	return keys, nil
 }
 
+// GetIndexKeysWithPagination returns keys matching composite key values from the chaincode db
+func (db *LedgerDB) GetIndexKeysWithPagination(index string, attributes []string, pageSize int32, bookmark string) ([]string, string, error) {
```

Comment:
May be minor but why passing pageSize as a parameter if you always use the same global.

I would propose to pass it during the initialization of the `LedgerDB` to make it explicit enough but still limit the number of parameters to a minimum.

Reply:
Not exactly.
If you look [here](https://github.com/SubstraFoundation/substra-chaincode/pull/131/files#diff-95f619a253f6926eaeb3deb37d0536e95f0418a7ecdb444145fb2080bfecfcf5R133) you can see that we not always use the same global everywhere. For this case, as we do multiple calls, we need to divide it by 3.

Reply:
Ok.